stable/luigi/Chart.yaml

```diff
@@ -18,5 +18,5 @@ keywords:
 - spark
 - kubernetes job manager
 name: luigi
-version: 2.7.4
+version: 2.7.5
```

Comment:
```suggestion
version: 2.8.0
```

Reply:
Thanks, changed to bump minor version in current commit (with squash)